### ISSUE TITLE
SyntaxNet : Not found: brain_pos/greedy/0/label-map

### ISSUE THREAD

**Issue by dsindex** [tensorflow/models#98]
I'd like to train pos tagger with `syntaxnet/testdata/mini-training-set`. 

- edit syntaxtnet/testdata/context.pbtxt
```
Parameter {
  name: 'brain_pos_transition_system'
  value: 'tagger'
}
Parameter {
  name: 'brain_pos_embedding_dims'
  value: '64;4;8;8'
}
Parameter {
  name: 'brain_pos_features'
  value: 'stack(3).word stack(2).word stack(1).word stack.word input.word input(1).word input(2).word input(3).word;input.digit input.hyphen;stack.suffix(length=2) input.suffix(length=2) input(1).suffix(length=2);stack.prefix(length=2) input.prefix(length=2) input(1).prefix(length=2)'
}
Parameter {
  name: 'brain_pos_embedding_names'
  value: 'words;other;suffix;prefix'
}
....
input {
  name: 'training-corpus'
  record_format: 'conll-sentence'
  Part {
    file_pattern: 'syntaxnet/testdata/mini-training-set'
  }
}
input {
  name: 'tuning-corpus'
  record_format: 'conll-sentence'
  Part {
    file_pattern: 'syntaxnet/testdata/mini-training-set'
  }
}
input {
  name: 'dev-corpus'
  record_format: 'conll-sentence'
  Part {
    file_pattern: 'syntaxnet/testdata/mini-training-set'
  }
}
....
input {
  name: 'label-map'
  creator: 'brain_pos/greedy'
}
input {
  name: 'word-map'
  creator: 'brain_pos/greedy'
}
input {
  name: 'lcword-map'
  creator: 'brain_pos/greedy'
}
input {
  name: 'tag-map'
  creator: 'brain_pos/greedy'
}
input {
  name: 'category-map'
  creator: 'brain_pos/greedy'
}
input {
  name: 'prefix-table'
  creator: 'brain_pos/greedy'
}
input {
  name: 'suffix-table'
  creator: 'brain_pos/greedy'
}
input {
  name: 'tag-to-category'
  creator: 'brain_pos/greedy'
}
...
``` 

- run command
```
$ pwd
/root/syntaxnet/models/syntaxnet
$ bazel-bin/syntaxnet/parser_trainer \
  --task_context=syntaxnet/testdata/context.pbtxt \
  --arg_prefix=brain_pos \  # read from POS configuration
  --compute_lexicon \       # required for first stage of pipeline
  --graph_builder=greedy \  # no beam search
  --training_corpus=training-corpus \  # names of training/tuning set
  --tuning_corpus=tuning-corpus \
  --output_path=models \  # where to save new resources
  --batch_size=32 \       # Hyper-parameters
  --decay_steps=3600 \
  --hidden_layer_sizes=128 \
  --learning_rate=0.08 \
  --momentum=0.9 \
  --seed=0 \
  --params=128-0.08-3600-0.9-0  # name for these parameters
```

- then, i got the error message.
```
F syntaxnet/term_frequency_map.cc:62] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: brain_pos/greedy/0/label-map)
```

how can i edit `context.pbtxt` correctly?

**Comment by calberti** [comment]
When you paste the parser_trainer command, can you make sure that the comments are removed and that the backslash is the last character on each line?  Otherwise your command should work.

```shell
bazel-bin/syntaxnet/parser_trainer \
  --task_context=syntaxnet/testdata/context.pbtxt \
  --arg_prefix=brain_pos \
  --compute_lexicon \
  --graph_builder=greedy \
  --training_corpus=training-corpus \
  --tuning_corpus=tuning-corpus \
  --output_path=models \
  --batch_size=32 \
  --decay_steps=3600 \
  --hidden_layer_sizes=128 \
  --learning_rate=0.08 \
  --momentum=0.9 \
  --seed=0 \
  --params=128-0.08-3600-0.9-0
```

**Comment by dsindex** [comment]
haha~ i did not suspect comments~ thank you :)
